class-wc-calypso-bridge.php

```diff
@@ -46,421 +41,172 @@ class WC_Calypso_Bridge {
+		require_once WC_CALYSPO_BRIDGE_PLUGIN_PATH . '/includes/class-wc-calypso-bridge-plugins.php';
+		require_once WC_CALYSPO_BRIDGE_PLUGIN_PATH . '/includes/class-wc-calypso-bridge-addons.php';
 	}
 
 	/**
-	 * Add styles for ecommerce plan.
+	 * Load ecommerce plan UI changes.
 	 */
-	public function add_ecommerce_plan_styles() {
-		$asset_path = self::$plugin_asset_path ? self::$plugin_asset_path : self::MU_PLUGIN_ASSET_PATH;
-		wp_enqueue_style( 'wp-calypso-bridge-ecommerce', $asset_path . 'assets/css/ecommerce.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION );
+	public function load_ecommerce_plan_ui() {
 
-		if ( (bool) apply_filters( 'ecommerce_new_woo_atomic_navigation_enabled', true ) ) {
-			wp_enqueue_style( 'wp-calypso-bridge-ecommerce-navigation', $asset_path . 'assets/css/ecommerce-navigation.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION );
-		}
-	}
+		/**
+		 * Disable block editor for post types.
+		 *
+		 * @param bool    $value
+		 * @param string  $post_type
+		 * @return bool
+		 */
+		add_filter( 'use_block_editor_for_post_type', static function( $value, $post_type ) {
+
+			$wc_post_types = array(
+				'shop_coupon',
+				'shop_order',
+				'product',
+				'bookable_resource',
+				'wc_booking',
+				'event_ticket',
+				'wc_membership_plan',
+				'wc_user_membership',
+				'wc_voucher',
+				'wc_pickup_location',
+				'shop_subscription',
+				'wc_product_tab',
+				'wishlist',
+				'wc_zapier_feed',
+			);
 
-	/**
-	 * Class instance.
-	 */
-	public static function instance() {
-		if ( is_null( self::$instance ) ) {
-			// If this is a traditionally installed plugin, set plugin_url for the proper asset path.
-			if ( file_exists( WP_PLUGIN_DIR . '/wc-calypso-bridge/wc-calypso-bridge.php' ) ) {
+			if ( in_array( $post_type, $wc_post_types ) ) {
+				return false;
+			}
 
-				if ( WP_PLUGIN_DIR . '/wc-calypso-bridge/' == plugin_dir_path( __FILE__ ) ) {
-					self::$plugin_asset_path = plugin_dir_url( __FILE__ );
-				}
+			return $value;
+		}, 10, 2 );
+
+		/**
+		 * Decalypsoify ecommerce plans in case the user meta has already been previously set. Remove calypsoify styles to prevent styling conflicts.
+		 *
+		 * @param null   $null Always null.
+		 * @param int    $object_id Object ID.
+		 * @param string $meta_key Meta key.
+		 * @return null|bool
+		 */
+		add_filter( 'get_user_metadata', function( $null, $object_id, $meta_key ) {
+			if ( 'calypsoify' === $meta_key ) {
+				return false;
 			}
 
-			self::$instance = new self();
-		}
+			return $null;
+		}, 10, 3 );
 
-		return self::$instance;
+		/**
+		 * Remove admin footer text.
+		 */
+		add_filter( 'woocommerce_display_admin_footer_text', '__return_false' );
 	}
 
 	/**
-	 * Remove WooCommerce footer text
+	 * Loads language files for the plugin.
+	 *
+	 * @since 2.0.0
 	 */
-	public function remove_woocommerce_core_footer_text() {
-		add_filter( 'woocommerce_display_admin_footer_text', '__return_false' );
+	public function load_transalation() {
```

Comment:
Nit: this should be `load_translation()`.
```suggestion
	public function load_translation() {
```